src/mutils/namespace.py

```diff
@@ -68,7 +69,7 @@ def getFromSelection():
         # Catch any errors when running this command outside of Maya
         logger.exception(error)
 
-    return namespaces
+    return list(OrderedDict.fromkeys(namespaces))
```

Comment:
Also, I'm interested to know what the reason was behind this line. `getFromDagPaths` should return a list of unique namespaces. Would it also make sense to add the same sort by logic that you added to matchnames.py to `getFromDagPaths`?

Reply:
Here is a cheat way to get rid of any duplicates and keep the order. It could be better with similar logic as in `matchnames.py`

Reply:
Yea, i think adding the sort logic from `matchnames.py` to `getFromDagPaths` makes the most sense. The other changes look good. thanks again for adding this.